jekyll/_cci2/orb-author.md

```diff
@@ -9,279 +9,289 @@ version:
+
+### Writing Your Orb
+
+Before you begin working on your orb, ensure you are on a non-default branch. We typically recommend starting your orb on the `alpha` branch.
+
+```shell
+$ git checkout -b alpha
 ```
-{% endraw %}
 
-In the example above, note that the contents of `my-orb` are resolved as an inline orb because the contents of `my-orb` are a map; whereas the contents of `codecov` are a scalar value, and thus assumed to be an orb URI.
+Once you have ran the `circleci orb init` command, you will have a repository with importantly, a `.circleci` and `src` directory.
+
+_Example: Orb Project Structure_
 
-### Example Inline Template
+| type | name|
+| --- | --- |
+| 📁| [.circleci](https://github.com/CircleCI-Public/Orb-Project-Template/tree/master/.circleci) |
+| 📁| [.github](https://github.com/CircleCI-Public/Orb-Project-Template/tree/master/.github) |
+| 📁| [src](https://github.com/CircleCI-Public/Orb-Project-Template/tree/master/src) |
+| 📄| [.gitignore](https://github.com/CircleCI-Public/Orb-Project-Template/blob/master/.gitignore) |
+| 📄| [CHANGELOG.md](https://github.com/CircleCI-Public/Orb-Project-Template/blob/master/CHANGELOG.md) |
+| 📄| [LICENSE](https://github.com/CircleCI-Public/Orb-Project-Template/blob/master/LICENSE) |
+| 📄| [README.md](https://github.com/CircleCI-Public/Orb-Project-Template/blob/master/README.md) |
+{: class="table table-striped"}
 
-When you want to author an orb, you may wish to use this example template to quickly and easily create a new orb with all of the required components. This example includes each of the three top-level concepts of orbs. While any orb can be equally expressed as an inline orb definition, it will generally be simpler to iterate on an inline orb and use `circleci config process .circleci/config.yml` to check whether your orb usage matches your expectation.
+#### Orb Source
 
-{% raw %}
+
+Navigate to the `src` directory.
+
+_Example: Orb Project "src" Directory_
+
+| type | name|
+| --- | --- |
+| 📁| [commands](https://github.com/CircleCI-Public/Orb-Project-Template/tree/master/src/commands) |
+| 📁| [examples](https://github.com/CircleCI-Public/Orb-Project-Template/tree/master/src/examples) |
+| 📁| [executors](https://github.com/CircleCI-Public/Orb-Project-Template/tree/master/src/executors) |
+| 📁| [jobs](https://github.com/CircleCI-Public/Orb-Project-Template/tree/master/src/jobs) |
+| 📄| [@orb.yml](https://github.com/CircleCI-Public/Orb-Project-Template/blob/master/src/%40orb.yml) |
+{: class="table table-striped"}
+
+These files and directories above are the only "_required_" components, you may additionally see a `scripts` and `tests` folder in your project and we will go over those shortly. If your orb however does not need any `executors` or `jobs` for instance, these directories can be deleted.
```

Comment:
So they aren't "required"? Maybe we can say this is a standard setup, rather than these are the required sections.

Reply:
Ya that makes sense. 

"The directories listed above represented a type of orb component that may be included with your orb, with `@orb.yml` acting as the root of our orb. You may additionally..."

?

Reply:
👍